xtask/src/commands/test.rs

```diff
@@ -57,7 +57,14 @@ impl Test {
 
             eprintln!("Running tests with features: {}", features.join(", "));
             cargo!(
-                ["test", "--workspace", "--locked", "--no-default-features"],
+                [
+                    "test",
+                    "--workspace",
+                    "--locked",
+                    "--no-default-features",
+                    "-p",
+                    "\"apollo-router,apollo-router-core\""
```

Comment:
Not in default members so this will work almost perfectly  :+1: 

You do need to remove xtask from the default members though. (Not sure why it's there because we have an alias already in .config/cargo.toml)

```suggestion
                    "test",
                    "--locked",
                    "--no-default-features",
```